test/tests/functional/pbs_qmgr.py

```diff
@@ -41,12 +41,26 @@
 from ptl.lib.pbs_ifl_mock import *
 
 
+@tags('smoke')
```

Comment:
I am not sure about this, shouldn't this be like "regression" instead? If we keep "smoke" then while it will still run full regression of qmgr resource interface tests, along with newly added tests in SmokeTests, it Means new tests in SmokeTests is kinda redundant test if we are running tests in this suite?

So, our goal was to move qmgr resource interface tests from SmokeTest to this suite, add lightweight new tests in SmokeTest to compensate for remove tests, and mark moved tests as regression, right? Am I missing anything here?